src/main/loadscreen.js

```diff
@@ -1,6 +1,7 @@
 /* globals start */
 
 import $ from "jquery";
+import * as main from "main/main";
```

Comment:
You need to cut this, it's in a different bundle that this file loads. Causes main to execute twice.